deploy/deploy.go

```diff
@@ -72,6 +72,27 @@ func logCommand(cmd string, args ...string) error {
 	return c.Run()
 }
 
+// logCommandWithInput runs the specified command but records standard output
+// with log.Info and standard error with log.Warning. in is sent to
+// the standard input of the command.
+func logCommandWithInput(in []byte, cmd string, args ...string) error {
```

Comment:
It would be nice to have a test for this code.  You should be able to test all three with a shell script as your input and sh as your command though you would probably need to have some test vars:

```
var logInfo = log.Info
var logWarning = log.Warning
```
so you can capture the output that would be sent to the log package.

Reply:
done